lib/http.js

```diff
@@ -265,6 +267,7 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
     // not cookies
     cors(),
     validate({bodySchema: zcapSchema}),
+    helpers.verifyDigestHeaderValue,
```

Comment:
This should move up above the body validator. Let's make sure that what was sent was sent by the authenticated party before trying to validate it.

Reply:
Updated here be69e6c